bundles/org.openhab.binding.mail/src/main/java/org/openhab/binding/mail/internal/config/POP3IMAPConfig.java

```diff
@@ -23,4 +23,15 @@
 @NonNullByDefault
 public class POP3IMAPConfig extends BaseConfig {
     public int refresh = 60;
+    
+    public String debugPrint() {
+        StringBuffer buffer = new StringBuffer();
```

Comment:
```suggestion
        StringBuilder buffer = new StringBuilder();
```

Reply:
Easy enough, but please explain why you think this is needed.